event/main_listener.php

```diff
@@ -156,6 +156,18 @@ public function startup($event)
 
 	public function overwrite_template_vars($event)
 	{
+		if ($this->user->data['is_registered'] && !$this->user->data['is_bot'])
+		{
+			$u_my_contribs = $this->controller_helper->route('phpbb.titania.author', array(
```

Comment:
How often will you stab me if I use some kind of `str_replace($titania_config->phpbb_script_path, $titania_config->titania_script_path, ...);` here?